mmap+memset+fork.c

```diff
@@ -46,6 +48,8 @@ void parse_options (int argc, char *argv[]){
 		{"end", required_argument, 0, 'e'},
 		{"map_size", required_argument, 0, 'm'},
 		{"set_size", required_argument, 0, 'n'},
+                {"map_populate", required_argument, 0, 'u'},
+                {"map_anonymous", required_argument, 0, 'a'},
 		{0,0,0,0}
```

Comment:
Watch your tabs vs. spaces here.  The rest of the files use tabs, so replace these space indentations with tabs.